### PR TITLE
 migrated composite actions and scripts removed

### PR DESCRIPTION
**Pull request by ng-htec**
github-release, init-app-release removed from examples
gh-release,init-release and utils folders removed
scripts folder removed
artifacts-ci updated
example for artifact-ci updated

### CODE REVIEW

File: artifacts-ci/artifacts-ci.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      app_version:
+        description: "Application Version"
+        value: ${{ steps.repo_data.outputs.app_version }}
+
+jobs:
+  build_push:
+    name: 'Verify / Build / Test / Publish Docker image for applications packaged as docker artifact'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get repo data
```
Comment (jz-htec) [inline artifacts-ci/artifacts-ci.yml:32]: checkout step is missing

**Review by jz-htec — CHANGES_REQUESTED** [review]

File: artifacts-ci/artifacts-ci.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        required: false
+        default: 'false'
+    outputs:
+      app_version:
+        description: "Application Version"
+        value: ${{ steps.repo_data.outputs.app_version }}
+
+jobs:
+  build_push:
+    name: 'Verify / Build / Test / Publish Docker image for applications packaged as docker artifact'
```
Comment (jz-htec) [inline artifacts-ci/artifacts-ci.yml:29]: Too long name, should be simplified

File: artifacts-ci/artifacts-ci.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+name: 'Artifact CI'
```
Comment (jz-htec) [inline artifacts-ci/artifacts-ci.yml:1]: @ng-htec What do you think to move all reusable workflows in `workflows` directory?
The structure will be in that case something like:
workflows
  |-- artifact-ci.yml
  |-- ini-release.yml
  |-- sonarscan
        |-- dotnet.yml

File: artifacts-ci/artifacts-ci.yml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    name: 'Verify / Build / Test / Publish Docker image for applications packaged as docker artifact'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+          token:  ${{ inputs.github_token }}
+      - name: Get repo data
+        id: repo_data
+        uses: htec-infra/gha-utils/utils/repo-data@v0.1.0
```
Comment (jz-htec) [inline artifacts-ci/artifacts-ci.yml:38]: update `uses` link with the latest changes in gha-utils

**Review by jz-htec — CHANGES_REQUESTED** [review]

File: artifacts-ci/artifacts-ci.yml (hunk 3 above)
Comment (ng-htec) [reply artifacts-ci/artifacts-ci.yml:1]: That is a great idea! It will be applied in next commit

**Review by jz-htec — APPROVED** [review]

**Pull request merged**